libyul/Object.cpp

```diff
@@ -87,11 +87,11 @@ Json Data::toJson() const
 
 Json Object::toJson() const
 {
-	yulAssert(code, "No code");
+	yulAssert(hasCode(), "No code");
 
 	Json codeJson;
 	codeJson["nodeType"] = "YulCode";
-	codeJson["block"] = AsmJsonConverter(0 /* sourceIndex */)(*code);
+	codeJson["block"] = AsmJsonConverter(0 /* sourceIndex */)(code()->root());
```

Comment:
```suggestion
	codeJson["block"] = AsmJsonConverter(0 /* sourceIndex */)(m_code->root());
```
I guess here you don't need such indirection. You already have access to `m_code`.